### PR TITLE
Update docs, and make PR to `spiceai/docs` for new `openapi.json`.

### PR DESCRIPTION
**Pull request by Jeadie**
# 🗣 Description
 - Updates based on feedback in https://github.com/spiceai/docs/pull/698. 
 - Also include change that makes PR in https://github.com/spiceai/docs for new `openapi.json` in this repo.

### CODE REVIEW

File: .github/workflows/generate_json_schema.yml
Comment (Jeadie) [inline .github/workflows/generate_json_schema.yml]: This is the only change, remaining is just automatic formatting.

Comment (peasee) [reply .github/workflows/generate_json_schema.yml]: Will this work? I'm not sure if the token in this workflow will have write access to that repo.

**Comment by Jeadie** [comment]
Blocked on https://github.com/spiceai/docs/pull/698

File: .github/workflows/generate_json_schema.yml
Comment (phillipleblanc) [reply .github/workflows/generate_json_schema.yml]: It will not have access to the other repo, unfortunately

Comment (Jeadie) [reply .github/workflows/generate_json_schema.yml]: Removed

**Review by peasee — APPROVED** [review]

**Pull request merged**
